### DISCUSSION TITLE
import asset maintenance records

### DISCUSSION THREAD

**Discussion by King-GO** [snipe/snipe-it#8963] [General]
I try to migrate MS Access database to snipe-it.
if it possible to import maintenance records to system? (the default import type only provide: Assets/Accessories/Comsumables/Components/Licenses/Users/)

the historical maintenance records including,
1. Calibration records,  calibration items (pressure gauge, timer,etc., ) , specification, measurement values, approval field
2. Maintenance records, maintenance items/activities, 

any method or alternatives way to achieve that?

Thank you!

**Comment by rtorresz** [comment]
Hi!

There is any way to import assets maintenance using the console or GUI? I recently imported through the console all the assets but I know the user wants the maintenance to those. 

Any thoughts?

Thanks,

**Comment by uberbrady** [comment]
Hi folks - no, we don't currently have any way of importing asset maintenance records. You could certainly do it via raw MySQL, but a small mistake could cause big problems in the end, so you would have to be very careful - taking backups first, and carefully inspecting that the results work out how you want.
